client/src/luigi-client.js

```diff
@@ -37,6 +37,9 @@ class LuigiClient {
   getClientPermissions() {
     return lifecycleManager.getClientPermissions();
   }
+  sendCustomEventToCore(msg, data) {
+    return lifecycleManager.sendCustomEventToCore(msg, data);
+  }
   /**
```

Comment:
I think it would make sense to add this function
`addCustomEventListener(msg, data) {
    return lifecycleManager.addCustomEventListener(msg, data);
  }
`
to receive the messages in the client as well.
(also we have to add it in _luigi-client.d.ts_)

What do you think?